### DISCUSSION TITLE
Micro SD card extender problems with skr mini v3 board

### DISCUSSION THREAD

**Discussion by 3d-not-2d** [mriscoc/Ender3V2S1#831] [General]
I compiled some firmware using the Ender 3V2-SKRME3V3-BLUBL-LA-MPC configuration files. I enabled #define NO_SD_DETECT in the adv.h and 
#define HAS_SD_EXTENDER 1 in the .h config files. After flashing the new firmware to my board it works normally with just the sd in the board or nothing in it, but when I have the SD extender plugged in and I turn it on my screen just lights up blank and stays blank. Kinda at a loss for what to do next.

**Comment by ghost** [comment]
do you have patch? I am new user to SKR v3 and maybe we could dig together

**Reply by 3d-not-2d** [reply]
i have not figured out a way past this problem yet this is pretty much my first reply didn’t get any on like 5 sub reddits

**Comment by mriscoc** [comment]
Could be a EMI problem with the extender.

**Reply by 3d-not-2d** [reply]
interesting do you think it could be worth picking up a ferrite bead to put on the cable and trying that?
